### ISSUE TITLE
[Feat]: ui.Selection: support `scrollWhileDragging` feature when the element selection is in progress.

### ISSUE THREAD

**Issue by gsnalinkode** [clientIO/joint#1885]
### What happened?

- While element selection in canvas not getting horizontal scroll and select other elements
- Expected selection behaviour need in our application as below reference

[screen-capture.webm](https://user-images.githubusercontent.com/70625179/200863560-d49a6225-ec6c-4ed2-8d97-78eb84a40447.webm)

### Version

3.6.1

### What browsers are you seeing the problem on?

Chrome

### What operating system are you seeing the problem on?

Windows

**Comment by kumilingus** [comment]
It is a nice feature.

Right now, the `ui.Selection` is not aware of the existence of the`ui.PaperScroller`. I think we can allow the `paper` option to be set to a `ui.PaperScroller` instance (the same way we did with the `ui.Stencil`). Then we can enable the `scrollWhileDragging` feature for the `ui,.Selection` automatically.

```js
new ui.Selection({
   paper: paperScroller
});
```

In the meantime, please use this workaround (using the `ui.PaperScroller` undocumented API).

```js
paper.on('blank:pointerdown', (evt) => {
    selection.startSelecting(evt);
    const listener = new joint.mvc.Listener();
    listener.listenTo(paper, {
        'blank:pointermove': (evt, x, y) => {
            scroller.scrollWhileDragging(evt, x, y);
        },
        'blank:pointerup': (evt) => {
            scroller.stopScrollWhileDragging(evt);
            listener.stopListening();
        }
    });
});
```

**Comment by gsnalinkode** [comment]
- As per documentation https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#mvc  **new joint.mvc.Listener()** is available for version 3.6
- But my current version is 3.3.1 so is there any work around for same.

**Comment by kumilingus** [comment]
```js
paper.on('blank:pointerdown', (evt) =>  selection.startSelecting(evt));
paper.on('blank:pointermove', (evt, x, y) => scroller.scrollWhileDragging(evt, x, y));
paper.on('blank:pointerup', (evt) =>  scroller.stopScrollWhileDragging(evt));
```

**Comment by gsnalinkode** [comment]
- By using above solution selection for multi object 

> ```js
> paper.on('blank:pointerdown', (evt) =>  selection.startSelecting(evt));
> paper.on('blank:pointermove', (evt, x, y) => scroller.scrollWhileDragging(evt, x, y));
> paper.on('blank:pointerup', (evt) =>  scroller.stopScrollWhileDragging(evt));
> ```

**Comment by gsnalinkode** [comment]
- By using solution below able to do selection of multiple objects while scrolling 
 ```
  paper.on('blank:pointerdown', (evt) =>  selection.startSelecting(evt));
  paper.on('blank:pointermove', (evt, x, y) => scroller.scrollWhileDragging(evt, x, y));
  paper.on('blank:pointerup', (evt) =>  scroller.stopScrollWhileDragging(evt));
```
- But failing to select  for long distance object with in the canvas using above solution.

**Comment by kumilingus** [comment]
This might be related to a bug that has been fixed in `v3.6` (see [changelog](https://changelog.jointjs.com/rappid/3.6.0/changelog)).

> **ui.Selection** - fix lasso coordinates while scrolling
Keep the lasso frame under the pointer regardless of how the user scrolls the paper container.

**Comment by kumilingus** [comment]
Implemented in JJ+ `v3.7`.
